### PR TITLE
epsilonitalia

### PR DESCRIPTION
**Pull request by fabiovinci**

### CODE REVIEW

File: epsilonitalia/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Contact
+This space is administered by:  
+
+**Fabio Vinci**  
+*INSPIRE/SDI Senior expert*  
+[Epsilon Italia srl](https://www.epsilon-italia.it)  
+Mendicino, Italy  
+<f.vinci@epsilon-italia.it>  
+GitHub: [nicholascar](https://github.com/fabiovinci)
```
Comment (TallTed) [inline epsilonitalia/README.md:15]: Either 
```suggestion
GitHub: [nicholascar](https://github.com/nicholascar)  
```

Comment (TallTed) [reply epsilonitalia/README.md:15]: or
```suggestion
GitHub: [fabiovinci](https://github.com/fabiovinci)  
```

Comment (TallTed) [reply epsilonitalia/README.md:15]: I believe it's meant to be the latter.

**Comment by dgarijo** [comment]
Thanks @TallTed, this has been fixed in another PR, I believe.

**Comment by fabiovinci** [comment]
Thanks to all for the review.
I fixed it in another PR.

**Pull request merged**